service/core/kafka/client/src/main/scala/com/lightbend/lagom/internal/broker/kafka/KafkaSubscriberActor.scala

```diff
@@ -40,6 +40,7 @@ private[lagom] class KafkaSubscriberActor[Message](
 
   override def receive: Actor.Receive = {
     case Status.Failure(e) =>
+      log.error("Topic subscription interrupted due to failure", e)
```

Comment:
+1

It is unlikely that the error is ambiguous and the stackTrace should provide enough info but will help `grep`-ping logs.